core/domain/email_manager.py

```diff
@@ -2188,8 +2190,11 @@ def send_email_to_new_contribution_reviewer(
     if review_category in [
             constants.CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION,
             constants.CONTRIBUTION_RIGHT_CATEGORY_REVIEW_VOICEOVER]:
-        # Ruling out the possibility of None for mypy type checking.
-        assert language_code is not None
+        if language_code is None:
+            raise Exception(
+                'The language_code cannot be None if the review category is'
+                ' translation or voiceover'
```

Comment:
Add `\'` quotation marks, as described in the previous comment.

Reply:
Done